plugins/auth-backend/src/providers/onelogin/provider.ts

```diff
@@ -120,7 +120,7 @@ export class OneLoginProvider implements OAuthHandlers {
       await executeRefreshTokenStrategy(
         this._strategy,
         req.refreshToken,
-        req.scope,
+        'openid',
```

Comment:
from [this comment it seems alright that this is hardcoded](https://github.com/backstage/backstage/issues/21776#issuecomment-1952173694).